offices/_single.html

```diff
@@ -1,8 +1,11 @@
 {% extends "layout-side-nav.html" %}
 {% import "_vars-offices.html" as vars with context %}
 
+{% set office_title = office.title | safe if office.title %}
+{% set office_tags  = office.tags if office.tags %}
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/6becc8400665d1ecf4897bde89636b41e03b27f5